GDJS/Runtime/scenestack.ts

```diff
@@ -354,5 +354,12 @@ namespace gdjs {
 
       return hasMadeChangeToStack;
     }
+
+    dispose(): void {
+        for (let i = 0; i < this._stack.length; ++i) {
+            this._stack[i].unloadScene();
+        }
```

Comment:
Use `for...of` or `forEach` because there is no logic with `i` involved.

Reply:
done